### DISCUSSION TITLE
ME query doesn't show all customer's orders

### DISCUSSION THREAD

**Discussion by DidoMarchet** [saleor/saleor#11623] [General] [answered]
Hi guys,
 I would like to know if there are some limits to the query:
```
me{
 orders: { .... }
}
```
because it doesn't show me all orders.
It shows ~53 orders but my account has more orders.

I didn't see any parameter  to filter or  paginate order in me query.

Thanks in advance and kind regards,

Davide

**Comment by DidoMarchet** [comment]
Hi guys,
In addition to the foregoing I would like to know if it is possible to filter in some way the order getted in the me query. For example we need to filter them by channel.

Thanks in advance and kind regards,

Davide

**Reply by Skitus** [reply]
@DidoMarchet Can you please show us request, cause I always get {me: NULL}

**Comment by peelar** [comment, marked as answer]
Hello @DidoMarchet 👋 

> I didn't see any parameter to filter or paginate order in me query.

The `orders` query accepts `first`, `last`, `before`, and `after`. Can you see if it does affect the returned results?

>  I would like to know if it is possible to filter in some way the order getted in the me query.

I am afraid the `orders` query in `me` only accepts the abovementioned parameters.

**Reply by DidoMarchet** [reply]
Hi @peelar sorry for the question I was a little confused and checking I see it's all right.
Ok for fitlering channel.

Thanks and kind regards,

Davide
